src/containers/List/List.tsx

```diff
@@ -245,7 +245,7 @@ export const List = ({
     filterItemsQuery,
     {
       variables: filterPayload(),
-      fetchPolicy: 'cache-and-network',
```

Comment:
Not sure how this is related to responsiveness?

Reply:
When we are opening or closing the sideDrawer this query is getting called again if we have this fetch-policy. I have checked multiple scenarios like adding deleting or editing a list item and there are no issues caused by changing this policy.